runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -458,6 +452,31 @@ public void onChatMessage(ChatMessage event)
 		{
 			removeGameTimer(STAFF_OF_THE_DEAD);
 		}
+
+		if (config.showFreezes() && event.getMessage().equals("<col=ef1020>You have been frozen!</col>"))
+		{
+			createGameTimer(ICERUSH, false);
+			createGameTimer(ICEBURST, false);
+			createGameTimer(ICEBLITZ, false);
+			createGameTimer(ICEBARRAGE, false);
+		}
+	}
+
+	@Subscribe
+	public void onGameTickChanged(GameTick event)
```

Comment:
This is wrong naming for the method

Reply:
`onGameTick`? If you are going to say its wrong could at least point me in the right direction? Saying what you said vs. "Change it to _______" takes the same amount of characters and is more productive.

Reply:
Yes, `onEventName` is the convention. (almost) All existing examples should be in that format as well.

Reply:
There is like 137 events that are wrongly named, but otherwise yeah :D , on + `EventClassName`